src/components/_common/illustration/IllustrationStatic.tsx

```diff
@@ -30,12 +28,12 @@ export const IllustrationStatic = ({
 
     return (
         <div
-            className={classNames(bem('image'), className)}
+            className={classNames(style.image, className)}
             aria-hidden="true"
             role="presentation"
         >
             <div
-                className={classNames(bem('icon'), bem('icon', 'icon1'))}
+                className={classNames(style.icon, 'icon1')}
```

Comment:
Finner ikke noe styling for icon1. Fjernes?

Reply:
Fjernet